ui/pages/confirmations/components/simulation-details/amount-pill.tsx

```diff
@@ -53,10 +54,18 @@ export const AmountPill: React.FC<{
   }
 
   if (asset.tokenId) {
-    const tokenIdPart = `#${hexToDecimal(asset.tokenId)}`;
+    const decimalTokenId = hexToDecimal(asset.tokenId);
+    const shortenedDecimalTokenId = shortenAssetId(decimalTokenId, 11, 4, 4);
 
-    amountParts.push(tokenIdPart);
-    tooltipParts.push(tokenIdPart);
+    const shortenedTokenIdPart = `#${shortenedDecimalTokenId}`;
+    const tooltipIdPart = `#${
+      shortenedDecimalTokenId === decimalTokenId
```

Comment:
Is this ternary doing anything? If they're the same, then it shouldn't matter which value we use?

Reply:
Right, I was tryign to simplify the first working version but missed this one removed in
https://github.com/MetaMask/metamask-extension/commit/7ae1d604fc3c9c559b2c4f32b62ad0e085184ea1